AutoDuty/Windows/Config.cs

```diff
@@ -246,6 +249,14 @@ public static void Draw()
             Configuration.Save();
         }
         ImGui.Separator();
+
+        if (ImGui.Checkbox("Using Alternative Rotation Plugin", ref Configuration.UsingAlternativeRotationPlugin))
```

Comment:
When did dalamud start letting us ref config's directly lol, I really need to pay more attention to change lolgs haha, good to know.

Reply:
Since always. You have to use fields instead of properties like I have.